app/uk/gov/hmrc/plasticpackagingtax/registration/views/group/organisation_list.scala.html

```diff
@@ -36,11 +36,7 @@
 
 @(form: Form[Boolean], nominatedOrganisation: String, members: Seq[GroupMember])(implicit request: JourneyRequest[AnyContent], messages: Messages)
 
-@numOrganisations = @{
-    {
-        members.size + 1
-    }.toString
```

Comment:
I like this; keeping the something which claims to be a number as a Int rather than a String :+1: